core/commons/che-core-commons-mail/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2017 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-core-commons-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>5.19.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>che-core-commons-mail</artifactId>
+    <packaging>jar</packaging>
+    <name>Che Core :: Commons :: Mail sender</name>
+    <dependencies>
```

Comment:
missing provided scope for API

Reply:
Are you talking about javax.mail?

Reply:
javax.inject, javax.annotation api, 
about mail usually only if an implementation is required we provide `mail` artifact  (that contains the implementation (like for the test scope)
else the `javax.mail-api` artifact is used for API calls